frontend/pulse3d/src/components/layouts/Layout.js

```diff
@@ -48,6 +48,10 @@ export default function Layout({ children }) {
     router.replace("/login", undefined, { shallow: true });
   };
 
+  const redirectHome = () => {
+    if (router.pathname !== "/login") router.replace("/uploads", undefined, { shallow: true });
```

Comment:
I think we want this to go to curibio.com, not the uploads table on dashboard.curibio.com